contributor_docs/es/contributor_guidelines.md

```diff
@@ -0,0 +1,62 @@
+
+### Usando la interfaz de línea de comandos de `git`
+
+Cuando hayas realizado el <em>fork</em> (copia), navega a la página de tu <em>fork</em> y copia la URL de git haciendo clic en el botón verde que dice “Code”. Debería ser algo así como `https://github.com/limzykenneth/p5.js.git`
+
+![Screenshot of the list of files on the landing page of a repository. The "Code" button is highlighted with a dark orange outline.](/contributor_docs/images/code-button.png)
+
+Después, ve a la línea de comandos de tu entorno local y clona este repositorio de git. “Clonar” simplemente significa descargar una copia del repositorio en tu máquina local. Corre el siguiente comando en un folder en donde quieras almacenar la carpeta del código fuente de p5.js. 
+
+```
+git clone [git_url]
+```
+
+Sustituye `[git_url]` por la URL acabas de copiar. Esto puede llevar varios minutos, dependiendo de la velocidad de tu conexión a internet, ¡un buen momento para preparar un poco de café! Una vez que el proceso finalice, puedes abrir la carpeta que descargaste llamada `p5.js` en tu editor de código preferido y comenzar a explorar. 
+
+## Desglose del código base
+
+Algunos de los archivos y carpetas clave que encontrarás en la carpeta de p5.js son los siguientes:
+
+- `src` - Aquí es donde reside todo el código que eventualmente se combina en los archivos finales p5.js y p5.min.js.
+- [`test`](https://github.com/processing/p5.js/blob/main/contributor_docs/unit_testing.md) - En esta carpeta se encuentran las pruebas unitarias y el código para probar todos los ejemplos de documentación.
+- `tasks` - Donde se encuentra el código de compilación detallado y personalizado. `Gruntfile.js` - Este es el archivo de configuración de compilación principal.
+- `contributor_docs` - Aquí es donde reside la documentación y toda la demás documentación para las, los y les colaboradores.
+
+Los demás archivos y carpetas son principalmente configuraciones u otros tipos de archivos de soporte; en la mayoría de los casos, no debería ser necesario realizar ninguna modificación.
+
```

Comment:
Falta la seccion de Build setup:
>Build setup
>Before you do anything, you'll need to set up the local project folder so that you can build and run tests for p5.js. Assuming you have node.js installed, run:
>
>npm ci
>This will likely take a while, as npm downloads all dependencies required. However, once done, that's it, you are all set up. Pretty simple, right?